ckcc/cli.py

```diff
@@ -835,6 +837,70 @@ def hsm_status():
         click.echo(pformat(o))
 
 
+@main.command('mins_ls')
```

Comment:
there should be a group command `ckcc minisc` with all related nested i.e. `ckcc minisc get` or `ckcc minisc delete`...

Reply:
done